src/engine_registry.hpp

```diff
@@ -516,19 +516,21 @@ namespace wr
 			rs_layout::Entry{(int)DoFCoCE::CAMERA_PROPERTIES, 1, rs_layout::Type::CBV_OR_CONST}
 		};
 
-		enum class DoFDownScaleE
+		enum class DownScaleE
 		{
 			SOURCE,
 			OUTPUT_NEAR,
 			OUTPUT_FAR,
+			OUTPUT_BRIGHT,
 			COC
 		};
 
-		constexpr std::array<rs_layout::Entry, 4> dof_down_scale = {
-			rs_layout::Entry{(int)DoFDownScaleE::SOURCE, 1, rs_layout::Type::SRV_RANGE},
-			rs_layout::Entry{(int)DoFDownScaleE::OUTPUT_NEAR, 1, rs_layout::Type::UAV_RANGE},
-			rs_layout::Entry{(int)DoFDownScaleE::OUTPUT_FAR, 1, rs_layout::Type::UAV_RANGE},
-			rs_layout::Entry{(int)DoFDownScaleE::COC, 1, rs_layout::Type::SRV_RANGE}
+		constexpr std::array<rs_layout::Entry, 5> down_scale = {
+			rs_layout::Entry{(int)DownScaleE::SOURCE, 1, rs_layout::Type::SRV_RANGE},
+			rs_layout::Entry{(int)DownScaleE::OUTPUT_NEAR, 1, rs_layout::Type::UAV_RANGE},
+			rs_layout::Entry{(int)DownScaleE::OUTPUT_FAR, 1, rs_layout::Type::UAV_RANGE},
+			rs_layout::Entry{(int)DownScaleE::OUTPUT_BRIGHT, 1, rs_layout::Type::UAV_RANGE},
```

Comment:
Downsacling shouldn't require 3 outputs.

Reply:
Im outputting for multiple passes mainly for DoF, the near output , far output and the brightness output for bloom. these passes require to be output separately to improve bokeh quality.